modules/nomad-cluster/main.tf

```diff
@@ -83,6 +83,7 @@ resource "aws_launch_configuration" "launch_configuration" {
     content {
       device_name           = ebs_block_device.value["device_name"]
       volume_size           = ebs_block_device.value["volume_size"]
+      snapshot_id           = ebs_block_device.value["snapshot_id"]
```

Comment:
Since `snapshot_id` is optional, this should use the `lookup` syntax, similar to the other optional items below.